.github/workflows/unit-tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Install python package
       shell: bash -l {0}
       run: |
-        python setup.py develop
+        pip install -e.
```

Comment:
This is my bad, but I think we need `pip install -e .[test]` to install everything needed for the unit tests.

Reply:
done